src/main/java/org/folio/rest/impl/UserTenantsAPI.java

```diff
@@ -87,6 +87,6 @@ private void addWhereClauseArgumentsToCriterion(String userId, String username,
     fields.entrySet().stream()
       .filter(entry -> StringUtils.isNotBlank(entry.getValue()))
       .map(entry -> new Criteria().addField(entry.getKey()).setOperation("=").setVal(entry.getValue()).setJSONB(false))
-      .forEach(criterion::addCriterion);
+      .forEach(a -> criterion.addCriterion(a, "or"));
```

Comment:
please add new optional param for this, if it skipped - "and" should be used by default and add unit tests